index.js

```diff
@@ -1,18 +1,56 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "orange"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+//1. Find the element that we're interested in
+const profile = document.querySelector('div#profile')
+
 
+const img = profile.querySelector('img')
+//2 .Actually update the property of that element
+img.src = traveler.photo
+img.alt = traveler.name
 
+const h2 = profile.querySelector('h2')
+h2.textContent = traveler.name
+
+const em = profile.querySelector('em')
+em.textContent = traveler.nickname
+
+console.log(em)
```

Comment:
Great! I like the variables to be indicative of their element. So something like:

```
const travelerDiv = document.querySelector('div#profile')

const travelerImg = travelerDiv.querySelector('img')
travelerImg.src = traveler.photo
travelerImg.alt = traveler.name

const travelerNameH2 = travelerDiv.querySelector('h2')
travelerNameH2.textContent = traveler.name 

const travelerNicknameEm = travelerDiv.querySelector('em')
travelerNicknameEm.textContent = traveler.nickname 
```

LOVE that you made sure to add the alt attribute for the image. Sylwia would be proud!